### PR TITLE
v5 docs: example shortcode regex

### LINKED ISSUE #29351: example shortcode: see if we can use a regex
https://github.com/twbs/bootstrap/blob/919b526f17a08cefba51c42b831c922adecb4a9f/site/layouts/shortcodes/example.html#L35-L38

The above basically extracts the class, removes `bd-placeholder-img` and `bd-placeholder-img-lg`.

I wonder if we can use `findRE` along with `replaceRE` to achieve the same thing.

Not a huge deal, it just feels like a more elegant solution.

Similarly we could try doing the same for:

https://github.com/twbs/bootstrap/blob/919b526f17a08cefba51c42b831c922adecb4a9f/site/layouts/shortcodes/example.html#L27-L29

This just extracts the `<svg></svg>`.

### PR DESCRIPTION
**Pull request by morrissey-ingenious**
example shortcode: see if we can use a regex

Fixes #29351

/cc @XhmikosR 

Preview: <https://deploy-preview-29377--twbs-bootstrap.netlify.com/>

### CODE REVIEW

File: site/layouts/shortcodes/example.html [hunk 1]
```diff
@@ -19,36 +19,10 @@
...
-        {{- else -}}
-          {{- $.Scratch.Add "highlight_content" `<img src="..." alt="...">` -}}
-        {{- end -}}
-      {{- else -}}
-        {{- $.Scratch.Add "highlight_content" $content_chunk -}}
-      {{- end -}}
-    {{- end -}}
-  {{- end -}}
+  
+  {{- $content := replaceRE `(class="\s*?")` `` (replaceRE `<svg.*class=.bd\-placeholder\-img(?:\-lg)?(?:\s*?bd\-placeholder\-img\-lg)?\s?(.*?)".*?<\/svg>\n` `<img src="..." class="$1" alt="..." >` $input) -}}
```
Comment (XhmikosR) [inline site/layouts/shortcodes/example.html]: We should probably split these 2 `replaceRE`s to make it clearer.

File: site/layouts/shortcodes/example.html [hunk 2]
```diff
@@ -19,36 +19,10 @@
...
+    
+  {{- $.Scratch.Set "highlight_content" $content -}}
```
Comment (XhmikosR) [inline site/layouts/shortcodes/example.html]: A simple variable assignment should work now, instead of Scratch. Can you try it?

**Comment by XhmikosR** [comment]
Thanks for the PR, I left a couple of comments.

I'll have a closer look in the next days :)

**Comment by XhmikosR** [comment]
BTW it seems this leaves a newline in some cases like https://deploy-preview-29377--twbs-bootstrap.netlify.com/docs/4.3/components/buttons/#examples

**Comment by morrissey-ingenious** [comment]
@XhmikosR : will do I just have been busy lately and haven't had time to revisit this.

**Comment by XhmikosR** [comment]
Cool, I thought you abandoned it :p

I can probably experiment too if I find some time sooner.

**Comment by morrissey-ingenious** [comment]
@XhmikosR: this has been updated

**Comment by XhmikosR** [comment]
It seems this results in an extra space, for example ``

**Comment by XhmikosR** [comment]
Thanks for the updates! I still haven't got the time to experiment with this closer, but it's in my TODO list.

**Comment by XhmikosR** [comment]
Alright, I pushed a few tweaks, let me know what you think @morrissey-ingenious.

**Comment by XhmikosR** [comment]
@morrissey-ingenious you overwrote my changes, please don't do that. See `git reflog` and reset this back to what it was please.

**Comment by XhmikosR** [comment]
I'm gonna close this and move it to an upstream branch where I control the branch changes.

Thanks for the patch and I'll ping you there @morrissey-ingenious

**Pull request closed without merging**